### DISCUSSION TITLE
Removing "Receive pre-release versions" does not allow official build to download.

### DISCUSSION THREAD

**Discussion by DustyMustard** [valinet/ExplorerPatcher#1051] [Q&A] [answered]
I was testing an issue with blue on the top taskbar and wanted to see if I could reproduce it when going from 22000.556.42.34 to official and assumed I could uncheck "recieve pre-release versions" and get it.

Granted I can uninstall (and will), but wanted to pass it on as this is the desired behavior with things when you leave the pre-release channel and try to update.

**Comment by valinet** [comment, marked as answer]
Yeah, a handful of folks solicited this behavior. They complained when they did unsubscribe from pre-release, the app was suggesting something older, indeed. So I specifically made it so it won’t downgrade you. So yeah, it’s by design now.

Maybe an option to “allow downgrades” or “always fetch latest version in a channel” to restore initial behavior would be good.
